cogs/destinyart.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, bot):
         self.bot = bot
         self.base_image_url = f'{CONFIG.Resources.image_bucket_root_url}/destiny-art/'
         self.num_poses = 51
-        self.num_dances = 54
+        self.num_dances = 55
```

Comment:
Small thing, but should eventually dynamically generate this number

Reply:
Indeed, this would require potentially indexing an endpoint in some manner.